rsocket-core/src/main/java/io/rsocket/fragmentation/FragmentationDuplexConnection.java

```diff
@@ -138,23 +138,9 @@ private ByteBuf encode(ByteBuf frame) {
     }
   }
 
-  private ByteBuf decode(ByteBuf frame) {
-    if (encodeLength) {
-      return FrameLengthFlyweight.frame(frame).retain();
-    } else {
-      return frame;
-    }
-  }
-
   @Override
   public Flux<ByteBuf> receive() {
-    return delegate
-        .receive()
-        .handle(
-            (byteBuf, sink) -> {
-              ByteBuf decode = decode(byteBuf);
-              frameReassembler.reassembleFrame(decode, sink);
-            });
+    return delegate.receive();
```

Comment:
@rstoyanchev here we remove reassembling logic since we always have it enabled

Reply:
This looks good although `ReassemblyDuplexConnection` could move up as  a base class of `FragmentationDuplexConnection`. That way you can have either re-assembly only or re-assembly and fragmentation in one.

Reply:
fixed